Answers.md

```diff
@@ -1,6 +1,11 @@
 ### Questions:
```

Comment:
Solid answers 👍 , perhaps in the future answer this questions as if you were explaining them to a 12 year old. It would help solidify your own knowledge of a subject and help explain yourself when having a conversation about the specific topic with someone who is not a Computer Scientist.